wazp/pages/02_ROI.py

```diff
@@ -1,21 +1,285 @@
 import dash
-from dash import html
+import dash_bootstrap_components as dbc
+import plotly.express as px
+from dash import dash_table, dcc, html
+from PIL import Image
 
-######################
-# Add page to registry
-#########################
+###############################
+# Add page to registry        #
+###############################
 dash.register_page(__name__)
 
-###############
-# Layout
-################
-layout = html.Div(
+###############################
+# Initialize variables        #
+###############################
+
+# Get initial video
+init_videos = ["No videos found yet"]
+# Get initial set of ROIs to initialize dropdown
+init_roi_names = ["No ROIs defined yet"]
+# Default color for ROI drawing
+init_roi_color = px.colors.qualitative.Dark2[0]
+# Initialize the frame slider parameters for each video
+init_frame_slider_params: dict = {"max": 1, "step": 1, "value": 0}
+init_frame_slider_storage: dict = {
+    v: init_frame_slider_params for v in init_videos
+}
+# Columns for ROI table
+init_roi_table_columns = ["name", "on frame", "path"]
+# Initialize the ROI storage dictionary
+init_roi_storage: dict = {v: {"shapes": []} for v in init_videos}
+# Initialize the ROI status alert
+init_roi_status: dict = {"message": "No ROIs to save.", "color": "light"}
+
+# Instructions for the user
```

Comment:
Some thoughts about the layout:
- It'd be nice if the layout was intuitive enough so that we can omit written instructions. If we see we do need them, one option to make them take less space could be to put them in a collapsable panel (just as we were thinking to show the code that generates plots).
- I find it more user friendly if all the inputs are on the same area. In this case, the selected video, the selected frame, and the ROI to label next. In that spirit I would suggest to move the 'Create new ROI for' just under the frame selector. Otherwise, I feel there is a lot of back and forth across the screen.
- similarly, I think the 'Save ROIs' button could be better placed under the table. After you define them, you review the table and then click save. I think it follows a bit more naturally the workflow. I'd keep the other buttons together with the 'Save ROIs' one, to have all the actions together.